microprofile.jdt/org.eclipse.lsp4mp.jdt.tp/org.eclipse.lsp4mp.jdt.target

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?>
+<target name="Running Platform">
+	<locations>
+		<location type="Target" uri="https://raw.githubusercontent.com/eclipse/eclipse.jdt.ls/master/org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target" />
+		<location includeSource="true" missingManifest="generate" type="Maven">
+			<feature id="io.smallrye.common.expression" label="Smallrye Commons Expression" provider-name="Smallrye" version="1.0.0.qualifier" />
```

Comment:
Is the `feature` element here really necessary ? All it seems to be doing is grouping the bundles, but it doesn't force the Bundle-SymbolicName or anything like that as far as I can see. Is it not enough to list all 4 of the below Maven artifacts under one `<dependencies>` element ?

Also, if you import and activate the target file in Eclipse IDE, you'll see it adds some elements to the XML (eg. `<type>jar</type>` inside <dependency> elements, and `includeDependencyDepth="none"` as an attribute of `<location>` (note that `infinite` is an option which is nice to know).

Reply:
I looked into this, it looks like by using `includeDependencyDepth="infinite"`, I can get rid of the references to the dependencies. I removed the `feature` element as well.

Reply:
Eclipse didn't do this automatically for me though?